app/src/main/java/ru/woodymsk/socialapp/domain/mapper/attachmentMapper.kt

```diff
@@ -0,0 +1,18 @@
+package ru.woodymsk.socialapp.domain.mapper
+
+import ru.woodymsk.socialapp.data.dto.AttachmentDTO
+import ru.woodymsk.socialapp.domain.model.AttachmentDAO
+import javax.inject.Inject
+
+class attachmentMapper @Inject constructor() {
+
+    fun mapToDao(attachment: AttachmentDTO) = AttachmentDAO(
+        attachment.url,
```

Comment:
```suggestion
        url = attachment.url,
```

ниже и в других мапперах пиши имена параметров. Будет у тебя 10 штук их, не поймешь, что происходит.

Reply:
в AttachmentMapper и PostMapper добавил все параметры